pkg/eventmonitor/config/config.go

```diff
@@ -0,0 +1,282 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package config
+
+import (
+	"fmt"
+
+	"os"
+
+	"strings"
+	"time"
+
+	"github.com/DataDog/datadog-agent/cmd/system-probe/config"
+	coreconfig "github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/ebpf"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+const (
+	rsNS = "runtime_security_config"
+	evNS = "event_monitoring_config"
+)
+
+type Config struct {
```

Comment:
I'm definitely with you on trying to avoid large config structures in general... this one is actually already a smaller version of what it used to be :) part of this change was to split one large config structure into 3 parts, and this is one of the parts. In this case, all these settings are used in the same place by the same object, so there isn't much of a reason to split it further at this time